_episodes/05-contribution-governance-communication.md

```diff
@@ -1,52 +1,380 @@
+grows you may need to create additional lists only for important announcements for instance.
+
+> ## Discussion: Develop communication strategies
+> #### Time: 5 minutes
+> Let's discuss what types of communication media do people prefer. Think about the team size,
 > and what would be most convenient way for people to communicate and get updates.
-> You have 5 min to discuss, in groups.
+>
+> > ## Solution
+> >
+> > Here is a list of recommendations. However, the efficient solutions will
+> > depend on the frequency of communication and the team size.
+> > - Community calls/meetings
+> > - Email list
+> > - Slack
+> > - Twitter
+> > - GitHub issues
+> > - Announcements (newsletters/updates)
+> > - Archive
+> {: .solution}
 {: .discussion}
 
-> ## `.testimonial`
-> “For me this experience has been incredible! Thank you for putting all your effort into this 
-> project. Many people responded with overflowing enthusiasm to participate and collaborate in the 
-> different facets of this project. It was a pleasure to work with all of you.” - H.S.
+## Recommendations
+
+The following checklist is only recommended key points to consider in making contributions easier, clear and transparent.
+
+> ## A checklist of things that can be included in your Contributing guidelines
+>
+> 1. Where can I contribute?
+> - User agreement
+> - Fixing typo
+> - Reporting and fixing bugs/errors
+> - Contributing to documentation
+> - Adding code/features
+> - Testing
+> - Writing papers
+> - Promoting
+> - Share ideas with the Community
+> - Organise events
+>
+> 2. How can I contribute?
+> - Technical requirements (software versions, space, etc)
+> - Skill requirements (programming language)
+> - Available resources
+> - Is there a style guide to refer to?
+> - What is allowed and not allowed
+> - What happens with my contributions
+> - Time frame that someone from the team responds to a contribution
+> - What if the contribution is accepted/ejected
+> - Citing your contributions (optional description of author, contributor, etc.)
+>
+> 3. Who should I contact?
+> - Main contact
+> - Owner
+> - Developer
+> - Maintainer
+> - Helper
+> - Tester
+> - Admin
+>
+> 4. What are the communication channels?
+> - Communication channels
+> - Announcements (newsletters, updates)
+> - Archive
+>
+> 5. What else should I know (links)?
+> - License
+> - Disclaimers
+> - Policies
+> - Code of conduct
+> - Data protection
+> - Main project page
+> - Main documentation
+> - Members/contributors page
+> - Resources to develop skills required for working on the project
+>
+> 6. What's in it for me?
+> - Develop new skills
+> - Find people with similar interests
+> - Find a mentor
+> - Teach others
+> - Learn how to work on open source projects
+>
+> 7. I don’t want to read all these, I only have a question!
+> - One contact person
+> - Minimal document
+{: .solution}
+
+## Nice examples of people contributing publicly
```

Comment:
those are example testimonials, not examples of people

Reply:
hi all, I submitted me review comments.

Reply:
thanks mateusz, I've address all the short reviews, will work on the longer ones with the group